vault_core/src/config.rs

```diff
@@ -90,9 +91,8 @@ impl Config {
         }
     }
 
-    pub fn get_epoch_from_slot(&self, slot: u64) -> Result<u64, VaultError> {
-        slot.checked_div(self.epoch_length())
-            .ok_or(VaultError::InvalidEpochLength)
+    pub fn get_epoch_from_slot(&self, slot: u64) -> Result<u64, CoreError> {
```

Comment:
remove this function and use get_epoch everywhere, no need for one line wrapper fn :)

Reply:
Done!